recipes/h5io/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "h5io" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "267af4fe374f287aa5e77ec386673ea22b02c6303c1485445a2b8c0a37b23f39" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
+    - h5py
+  run:
+    - python
+    - numpy
+    - h5py
+
+test:
+  imports:
+    - h5io
+
+about:
+  home: http://github.com/h5io/h5io
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Please add the `license_file`. Preferably it would come from the source distribution, but if not you can manually package it as described in:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

It might be good to raise an issue upstream to have it included in the source distribution and put a link to the issue url as a comment in this recipe so that if it is included in a future release, there is a visual indicator to remind you to check for future releases of the conda-forge package.

Reply:
It exists here:

https://github.com/h5io/h5io/blob/master/LICENSE.txt